lib/sda/host.go

```diff
@@ -656,3 +661,27 @@ func (h *Host) checkPendingTreeMarshal(el *EntityList) {
 	}
 	h.pendingTreeLock.Unlock()
 }
+
+// Testing helpers functions
+// NewHostMock returns a freshly generated host at this address with a random
+// keypair
+func NewHostMock(s abstract.Suite, address string) *Host {
```

Comment:
If this is intended to be used in tests only related to this package,  do not export it (make it 'private') or move it to the _test file. Same with `SetupHostMock`